docs/authentication/saml/overview.mdx

```diff
@@ -51,19 +51,15 @@ This will work: Once the user comes back from the IdP, they will need to go thro
 
 ### What happens if I delete the SAML connection? Will my users be deleted?
 
-The users will not be deleted, so your application will not break. However, they will need to “reintroduce” themselves to your new strategies by resetting their passwords or via OTP (depending on the strategy you choose).
+The users will not be deleted, so your application will not break. However, they will need to use another strategy (e.g. password, email OTP etc) to authenticate themselves moving forward.
```

Comment:
Remove the use of "e.g." as it may not be understood in other cultures/translations. 

```suggestion
The users will not be deleted, so your application will not break. However, they will need to use another [strategy](/docs/authentication/configuration/sign-up-sign-in-options#authentication-strategies), such as password, email OTP, etc., to authenticate themselves moving forward.
```